INSTRUCTION.md

```diff
@@ -0,0 +1,15 @@
+#Dockerhub
+repository https://hub.docker.com/repository/docker/deadmarvald/todoapp/
+
+
+#Build container
+docker build -t todoapp:1.0.0 .
+
+
+#Run container
+docker run -d -p 8080:8080 todoapp:1.0.0
```

Comment:
Make sure that the application inside the container is set to listen on port 8080, as this is the port being exposed and mapped to the host.